tests/network/general/test_ip_family_services.py

```diff
@@ -109,6 +109,7 @@ def expected_num_families_in_service(request, dual_stack_cluster):
 class TestServiceConfigurationViaManifest:
     @pytest.mark.polarion("CNV-5789")
     @pytest.mark.single_nic
+    # Not marked as `conformance`; requires NMState
```

Comment:
Same here, it shouldn't require nmstate, and I indeed don't see it used down the flow.